config/routes.rb

```diff
@@ -2,11 +2,18 @@
 
 require 'sidekiq/web'
 
+# Rails.application.routes.draw do
+#   authenticate :user do
+#     mount Sidekiq::Web => 'admins/sidekiq'
+#   end
+
 Rails.application.routes.draw do
-  authenticate :admin do
-    mount Sidekiq::Web => 'admins/sidekiq'
+  def draw(routes_name)
```

Comment:
use [THIS](https://github.com/mperham/sidekiq/issues/4061#issuecomment-629830410) instead